tests/page/page-route.spec.ts

```diff
@@ -520,7 +524,7 @@ it('should work with encoded server - 2', async ({ page, server, browserName })
   });
   const response = await page.goto(`data:text/html,<link rel="stylesheet" href="${server.PREFIX}/fonts?helvetica|arial"/>`);
   expect(response).toBe(null);
-  if (browserName === 'firefox')
+  if (browserName === 'firefox' || browserName === '_bidiFirefox')
     expect(requests.length).toBe(2); // Firefox DevTools report to navigations in this case as well.
```

Comment:
Is there a bug about it in Firefox? I'd expect only one fonts request here similar to the other browsers.

Reply:
Yes, there's https://bugzilla.mozilla.org/show_bug.cgi?id=1903274.
We could change the test so that it doesn't run into this Firefox bug by adding `<meta charset="utf-8">` to the data URL.